source/usage-examples/updateMany.txt

```diff
@@ -0,0 +1,113 @@
+===========================
+Update a Multiple Documents
+===========================
+
+.. default-domain:: mongodb
+
+You can update multiple documents using the ``updateMany()`` method on
+a ``MongoCollection`` object. The method accepts a **filter** that matches the
+document you want to update and an **update** statement that instructs the
+driver to change the matching document. The ``updateMany()`` method only
+updates all the documents in the collection that match the filter.
+
+To perform an update with the ``updateMany()`` method, you must pass
```

Comment:
```suggestion
To perform an update with the ``updateMany()`` method, you must provide
```
To me, "provide" implies that you should construct these yourself, while "pass" implies that these parameters might already exist and you just have to pass them along. But I admit that it's entirely subjective.

Reply:
They imply the same thing to me and both require qualifiers to discern whether it's a pre-existing object or if you're declaring them inline. In my own experience, I have encountered "pass parameters" more often than "provide parameters" in documentation and other forms of communication.

Reply:
Seems reasonable, more common = clearer for our readers.